ooniapi/services/oonirun/src/oonirun/routers/v2.py

```diff
@@ -51,6 +52,24 @@ class OONIRunLinkNettest(BaseModel):
         default=False, title="if this test should be enabled by default for manual runs"
     )
 
+    targets_name: Optional[str] = Field(
```

Comment:
I think we should do some validation on create and edit, that if `targets_name` is provided, you can't also provide `input` and `inputs_extra`, since these are to be populated based on the value of `targets_name`.

Reply:
is it an error to not provide any of them?

Reply:
It's not an error, because you can have a test that has no input for it (eg. `whatsapp`)